src/p2p-money/pending-payments.js

```diff
@@ -0,0 +1,73 @@
+// Scheduled checks on payment statuses
+
+module.exports = {
+
+  processReceivedCode: function(payment, code, senderTel) {
+    if (senderTel != payment.get("PayerMobile") && senderTel != payment.get("PayeeMobile")) {
+      throw new Error("Sender phone # is not associated with the payment specified.");
+    }
+
+    const isPayee = senderTel != payment.payerMobile;
+    // map these for legibility
+    let response = mapCodeToResponse(code);
+
+    if (isPayee) { // is runner or receiving balancer
+      switch (response) {
+        case "yes":
+          // Consider finalized - Update status to Completed
+          break;
+        case "no":
+          // Txt donor follow up (“Roger indicated that they have not received a venmo from you yet. If you can, please {venmo instructions}. If you are sure that you have already sent this, please respond to this message with “XYZB”. If you are unable to complete this donation, please respond with “!XYZB” (exclamation mark in front) so we can have someone else’s donation cover this expense. Thank you!”
+          // Update status to PendingDonorMessaged
+          break;
+        case "wait":
+          // shouldn't happen, consider sending back a "that didn't make any sense" and the full instructions again
+          break;
+      }
+    } else { // is paying donor or sending balancer
+      switch (response) {
+        case "yes":
+          // Donor says they sent
+          // Text runner to either confirm they received or visit {link} to dispute -- link should be to Disputes form, prefilled with donor payment ID
+          // Update status to DisputePendingNotes
+          break;
+        case "no":
+          // Donor not going to send
+          // Update status to failed
+          // Payment is re-enqueued
+          break;
+        case "wait":
+          // Change status to Pending, will automatically re-text runner in another X minutes
+          break;
+      }
+    }
+
+  },
+
```

Comment:
ehhhh, this is dumb; rather than run everything through here, let's break this up into individual airtable queries. still need to know how to schedule them.